salt/modules/flatpak.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+"""
+Manage flatpak packages via Salt
+
+.. versionadded:: Neon
+
+:depends: flatpak for distribution
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+import subprocess
+import re
+
+import salt.utils.path
+
+FLATPAK_BINARY_NAME = 'flatpak'
+
+__virtualname__ = 'flatpak'
+
+
+def __virtual__():
+    if salt.utils.path.which('flatpak'):
+        return __virtualname__
+
+    return False, 'The flatpak execution module cannot be loaded: the "flatpak" binary is not in the path.'
+
+
+def install(location, name):
+    """Install the specified flatpak package or runtime from the specified location.
```

Comment:
Salt uses single quotes for docstrings... and add a new line after the `'''`
```
'''
Install the speci....
...
```

Do this throughout the file